### PR TITLE
VP-6766: Update assigned roles blades UI

### PR DESCRIPTION
**Pull request by AliveMen**
Update role table style

### CODE REVIEW

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-roles-list.js [hunk 1]
```diff
@@ -28,22 +28,16 @@ angular.module('platformWebApp')
             return _.any(blade.currentEntities, function (x) { return x.$selected; });
         }
 
+        $scope.selectButtonClick = (row) => {
+            row.$selected = !row.$selected
+        };
+
```
Comment (eMazeika) [inline src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-roles-list.js:34]: Could this be done in template, not in the controller?

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/role-detail.js [hunk 2]
```diff
@@ -106,6 +107,7 @@ angular.module('platformWebApp').controller('platformWebApp.roleDetailController
         });
         blade.selectedAll = false;
     }
+    $scope.selectButtonClick = (row) => row.$selected = !row.$selected ;
```
Comment (eMazeika) [inline src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/role-detail.js:110]: Could this be done in template, not in the controller?

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-roles-list.js (hunk 1 above)
Comment (AliveMen) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-roles-list.js]: done

File: src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/role-detail.js (hunk 2 above)
Comment (AliveMen) [reply src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/role-detail.js]: done

**Review by eMazeika — APPROVED** [review]

**Comment by Vectorfield4** [comment]
Ready for merge.

**Pull request merged**